pkgs/tools/system/kanata/default.nix

```diff
@@ -0,0 +1,39 @@
+{ fetchFromGitHub
+, lib
+, libevdev
+, pkg-config
+, rustPlatform
+}:
+
+let
+  pname = "kanata";
+  version = "1.0.5";
+in
+rustPlatform.buildRustPackage {
+  inherit pname version;
+
+  src = fetchFromGitHub {
+    owner = "jtroo";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-sL9hP+222i8y0sK3ZEx66yXBTgZp5ewoPUlZS4XnphY=";
+  };
+
+  cargoHash = "sha256-uhN1UdwtU0C0/lpxUYoCcMLABFTPNO5wKsIGOBnFpzw=";
+
+  nativeBuildInputs = [ pkg-config ];
+
+  buildInputs = [ libevdev ];
+
+  meta =
+    let
+      inherit (lib) licenses maintainers platforms;
+    in
+    {
+      description = "A cross-platform advanced keyboard customization tool";
+      homepage = "https://github.com/jtroo/kanata";
+      license = licenses.lgpl3;
+      maintainers = with maintainers; [ linj ];
+      platforms = platforms.linux;
+    };
```

Comment:
```suggestion
  meta = with lib; {
    description = "A cross-platform advanced keyboard customization tool";
    homepage = "https://github.com/jtroo/kanata";
    license = licenses.lgpl3Plus;
    maintainers = with maintainers; [ linj ];
    platforms = platforms.linux;
  };
```

Reply:
Thanks for your review.
May I ask for the reason? I personally don't like `with`.

Reply:
1. Just to minimize the amount of text.
2. It's basically a de facto standard for Nixpkgs because it's written like this everywhere.

Reply:
with here is a total no brainer and the standard way to do things.